### DISCUSSION TITLE
throttle config.ini file

### DISCUSSION THREAD

**Discussion by dbryant69** [flybywiresim/aircraft#4317] [Help] [answered]
Just downloaded the latest installer and the new stand alone dev a32nx. Trying to re config my tca throttle quadrant and find that the throttle configuration.ini file that used to be in the work folder is missing. Has it moved elsewhere or can we now not change the settings to allow the throttle to work properly. The sensitivity settings all appear to be ok but when the throttle quadrant is set as idle, the a/c throttle is in full reverse.

**Comment by aguther** [comment, marked as answer]
Please use the EFB to configure you throttles. See also guide here:
https://www.youtube.com/watch?v=9oqFfAMXCcc
